rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] It would be better if you push the person and book to the rentals array you created earlier like this.
```
book.rentals << self
person.rentals << self
```

Reply:
Thank you! @Lucas-Masaba,
I really appreciate it.